crmsh/bootstrap.py

```diff
@@ -235,6 +235,22 @@ def wait_for_cluster():
     status_done()
 
 
+def pick_default_value(default_list, prev_list):
+    """
+    Provide default value for function 'prompt_for_string'.
+    Make sure give different default value in multi-ring mode.
+
+    Parameters:
+    * default_list - default value list for config item
+    * prev_list    - previous value for config item in multi-ring mode
+    """
+    if not default_list:
+        return ""
+    for value in default_list:
+        if value not in prev_list:
+            return value
+
```

Comment:
hi krig, for this function, I've tried to make it more simple:)
just make sure to give different default value in multi-ring config

e.g.
```
ring0[10.10.10.1] 20.20.20.1
ring1[10.10.10.1]
```
for ring0, user not pick the default ip "10.10.10.1", input the original second default ip "20.20.20.1",
so for ring1, if we still give the default value as "20.20.20.1", this will not work, the default value must always not be already configured

Reply:
Right now this could return `None` if `value` is not in `default_list`: It would be better to `return ""` as the last line in the function just in case :)